src/components/synopsis-report-form/synopsis-report-form.js

```diff
@@ -746,7 +746,7 @@ class SynopsisReportForm extends React.Component {
     const imageUploadJSX = (
       <React.Fragment>
         <div>
-          <h5 className={ this.state.ijImageOK ? '' : 'required' }>IDENTITY JOURNAL AND IMAGE UPLOAD</h5>
+          <h5 className={ this.state.ijImageOK ? '' : 'optional' }>IDENTITY JOURNAL AND IMAGE UPLOAD</h5>
```

Comment:
we don't have a class 'optional'.  You can just remove the className attribute from this h5.  Also, please add "(OPTIONAL)" to the title string.